src/api.js

```diff
@@ -0,0 +1,63 @@
+import { ListFacade } from './facades/lists';
+import { ListValidator } from './validators/lists';
+
+export const Api = {
+  async getList(options) {
+    ListValidator.listValidation(options);
+
+    return ListFacade.getList(options);
+  },
+
+  getAllLists(options) {
+    ListValidator.basicValidation(options);
+
+    return ListFacade.getAllLists(options);
+  },
+
+  getListItemCount(options) {
+    ListValidator.listValidation(options);
+
+    return ListFacade.getListItemCount(options);
+  },
+
+  insertItemOnList(options) {
+    ListValidator.itemValidation(options);
+
+    let items;
+    if (!options.skuList) {
+      items = [{ id: options.itemId }];
+    } else if (Array.isArray(options.skuList)) {
+      items = options.skuList
+        .filter(skuId => typeof skuId === 'string' || skuId instanceof String)
```

Comment:
ambos tem o mesmo valor, pois se o tipo do skuId for string ele vai ser instancia de string. Usa somente um